starter-code/server.js

```diff
@@ -1,27 +1,29 @@
 'use strict';
 
-// TODO: Initialize your project using NPM to create and populate a package.json file
-
-// TODO: Require the Express package that you installed via NPM, and instantiate the app
-// Remember to install express, and be sure that it's been added to your package.json as a dependency
-// There is also a package here called body-parser, which is required in for use in a new route.
-// Be sure to install that and save it as a dependency after you create your package.json.
+const express = require('express');
+const app = express();
 const bodyParser = require('body-parser').urlencoded({extended: true});
 const PORT = process.env.PORT || 3000;
 
-// TODO: Include all of the static resources as an argument to app.use()
-
-// TODO: (STRETCH) Write a new route that will handle a request and send the new.html file back to the user
+app.use(express.static('./public'));
 
+app.get('/new', function(req, res){
+  res.sendFile('/new.html', {root: './public'});
+});
 
 app.post('/articles', bodyParser, function(request, response) {
   // REVIEW: This route will receive a new article from the form page, new.html,
   // and log that form data to the console. We will wire this up soon to actually
   // write a record to our persistence layer!
   console.log(request.body);
   response.send('Record posted to server!!');
-})
+});
 
 app.listen(PORT, function() {
   // TODO: Log to the console a message that lets you know which port your server has started on
+  console.log('You are on port:', PORT);
+});
+
+app.use(function(req, res, next) {
```

Comment:
i guess it's for route handling after you get an error. I didn't end up using it but it was in all of the documentation so I decided to play it safe and leave it in. Just like how I've never used the req parameter.